lib/backend/k8s/resources/ippool.go

```diff
@@ -74,5 +74,11 @@ func (c IPPoolv1v3Converter) ConvertFromK8s(inRes Resource) (Resource, error) {
 		ipp.Spec.IPIP = nil
 	}
 
+	// Take a logical OR of the v1 NATOutgoing field with the v3 NATOutgoing.
+	ipp.Spec.NATOutgoing = ipp.Spec.NATOutgoingV1 || ipp.Spec.NATOutgoing
```

Comment:
is `ipp.Spec.NATOutgoing |= ipp.Spec.NATOutgoingV1` valid shorthand.... i forget

Reply:
Should really say that since we omit the field from the JSON/YAML if set to false then if either the v1 or v3 value are true then we know NATOutgoing should be enabled.

Reply:
nope